pkgs/applications/science/math/maxima/default.nix

```diff
@@ -70,10 +70,6 @@ stdenv.mkDerivation ({
       url = "https://git.sagemath.org/sage.git/plain/build/pkgs/maxima/patches/maxima.system.patch?id=07d6c37d18811e2b377a9689790a7c5e24da16ba";
       sha256 = "18zafig8vflhkr80jq2ivk46k92dkszqlyq8cfmj0b2vcfjwwbar";
     })
-    # There are some transient test failures. I hope this disables all those tests.
-    # If those test failures ever happen in the non-ecl version, that should be
-    # reportetd upstream.
-    ./known-ecl-failures.patch
```

Comment:
Why did you remove that patch?

Reply:
My impression: ECL Maxima is only worth the effort with Sage, so the ECL-specific code can be just present in the version used for Sage. As upstream changes in both Maxima and Sage (that might settle on some patches for Maxima before it is released upstream) happen, we can react to them accordingly.

Reply:
I see. That makes sense.

Reply:
After the update the patch didn't apply anymore, which broke `maxima-ecl`. Since all tests are now disabled anyway, I saw little point in putting in the effort of rebasing it.